resources/views/components/organisms/_o-table-of-contents.blade.php

```diff
@@ -1,21 +1,30 @@
 <div class="o-accordion{{ (isset($variation)) ? ' '.$variation : '' }}" data-behavior="accordion">
     @foreach ($items as $item)
+        @php
+            if (isset($currentArticle)) {
+                $isExpanded = $currentArticle->parent === $item || $currentArticle->present()->isArticleInTree($item->children);
+            } else {
+                $isExpanded = true;
+            }
+            $isActive = isset($currentArticle) && $item->id === $currentArticle->id;
```

Comment:
This can be simplified by pre-setting default values:

```
$isExpanded = true;
$isActive = false;
if (isset($currentArticle)) {
    $isExpanded = $currentArticle->parent === $item || $currentArticle->present()->isArticleInTree($item->children);
    $isActive = $item->id === $currentArticle->id;
}
```